air-quality-backend/api/src/mappers/texture_mapper.py

```diff
@@ -0,0 +1,27 @@
+from datetime import UTC
+from typing import List
+
+from src.types import TextureDto
+from shared.src.database.forecasts import DataTexture
+
+
+def database_to_api_result(measurement: DataTexture) -> TextureDto:
+
+    relative_uri = measurement["texture_uri"].replace("/app/data_textures/", "")
+    # absolute_uri = f'{os.getenv("UI_URL")}/{relative_uri}'
+
+    return {
+        "base_time": measurement["forecast_base_time"].astimezone(UTC),
+        "variable": measurement["variable"],
+        "time_start": measurement["time_start"],
+        "time_end": measurement["time_end"],
+        "chunk": measurement["chunk"],
+        "source": measurement["source"],
+        "texture_uri": f"/{relative_uri}",
+        "min_value": measurement["min_value"],
+        "max_value": measurement["max_value"],
+    }
+
+
+def map_texture(measurements_from_database: List[DataTexture]) -> List[TextureDto]:

```

Comment:
For reasons I don't understand it appears we aren't checking coverage of the API, but I think we're missing unit tests for this file?

Reply:
added unit test file `texture_mapper_test.py`